rest/dep_custom.go

```diff
@@ -113,3 +139,27 @@ func (s *Server) getInputNameFromCustom(deploymentID, nodeName, interfaceName, c
 	}
 	return result, nil
 }
+
+// checkInstances checks if provided instances exist and returns true in this case.
+// If a provided instance does not exists, returns false and the inexistent instance name
+func (s *Server) checkInstances(deploymentID, nodeName string, provInstances []string) (bool, string) {
+	// Get known instances for the provided node
+	allInstances, err := deployments.GetNodeInstancesIds(s.consulClient.KV(), deploymentID, nodeName)
+	if err != nil {
+		log.Panic(err)
+	}
+	// Check that provided instances exist
+	for _, provInstance := range provInstances {
+		var found bool
```

Comment:
You can make it simpler here with collections.ContainsString

Reply:
+1